core/grants.go

```diff
@@ -47,7 +47,7 @@ func (a *API) createGrant(ctx context.Context, x apiGrant) error {
 	// so we should do our equality comparisons accordingly.
 	guardData, err := json.Marshal(x.GuardData)
 	if err != nil {
-		return errors.Wrap(err)
+		return nil, errors.Wrap(err)
 	}
 
 	g := authz.Grant{
```

Comment:
I want to return the grant including the `CreatedAt` timestamp, which StoreGrant generates or retrieves.